teuthology/nuke/__init__.py

```diff
@@ -277,7 +277,7 @@ def nuke(ctx, should_unlock, sync_clocks=True, noipmi=False):
 def nuke_one(ctx, target, should_unlock, synch_clocks,
              check_locks, noipmi):
     ret = None
-    keep_logs = ctx.keep_logs
+    keep_logs = getattr(ctx, 'keep_logs', False)
```

Comment:
Since keep_logs, is nuke cli specific, why just don't add this as an optional argument `keep_logs=False` similar to other parameters like `noimpi` and `sync_clocks`?

Reply:
sure we can do that, don't want to end up with too many args for the function but one more doesn't hurt